lib/vsc/administration/ldapsync.py

```diff
@@ -197,11 +197,15 @@ def sync_altered_groups(self, last, dry_run=True):
                 # if a 404 occured, the group is not an VO, so we skip this. Otherwise something else went wrong.
                 if err.code != 404:
                     raise
+            group_moderators = [str(m) for m in group.moderators]
+            institute_name = str(group.institute['name'])
+            if not group_moderators:
```

Comment:
shouldn't this be done at the level of the accountpage?

Reply:
the group has no moderators, and that what's in the AP. we need a non-empty set of moderators for the ldap schema, that's why we will inject this here.
i prefer to keep the AP data clean and truthful

Reply:
What @stdweird said.